init.lua

```diff
@@ -8,8 +8,10 @@ local pvp_areas_modname = minetest.get_current_modname()
 
 local hasareasmod = minetest.get_modpath("areas")
 
-local safemode = minetest.setting_getbool("pvp_areas.safemode")
+local safemode = minetest.setting_getbool("pvp_areas.safemode") or false
 local area_label = minetest.setting_get("pvp_areas.label") or "Defined area."
+-- if false Mob does Damage
+local mobsDoNoDamage = false
```

Comment:
If `mobs do no damage` is false, that should mean that `mobs do damage`. Incoherent logic.

Instead, assign `KILL_OK` or `KILL_NO` to avoid confusion.

Also, since the boolean return for `on_hit()` is not always intuitive, it would be better to call the variable `mobs_hit_mode`